ClientApp/src/components/Authentication/AuthService.js

```diff
@@ -0,0 +1,98 @@
+import decode from 'jwt-decode';
+
+export default class AuthService {
+  // Initializing important variables
+  constructor(domain) {
+    this.domain = domain || 'http://localhost:8080'; // API server domain
```

Comment:
You should probably consider getting the current domain name  that is being used. The reason being we can have local and prod (capstone1.azurewebsites.net).

Reply:
also our api endpint is always in the following format /api/{controller}/{action}  In this case (/api/auth/login) - Have a look int he AuthController and make the required changes.